File: 2018-06-13/W2D3/W2D3/SecondViewController.m
//
//  SecondViewController.m
//  W2D3
//
//  Created by Jason Liang on 6/13/18.
//  Copyright © 2018 jasonliang. All rights reserved.
//

#import "SecondViewController.h"

@interface SecondViewController ()

@end

@implementation SecondViewController

- (void)viewDidLoad {
  [super viewDidLoad];
  // Do any additional setup after loading the view, typically from a nib.
  
  UINib *nib = [UINib nibWithNibName:@"CustomView" bundle:[NSBundle mainBundle]];
  NSArray *views = [nib instantiateWithOwner:nil options:nil];
  UIView *customView = [views firstObject];
  [self.view addSubview:customView];
  
  customView.translatesAutoresizingMaskIntoConstraints = NO;
}


- (void)didReceiveMemoryWarning {
  [super didReceiveMemoryWarning];
  // Dispose of any resources that can be recreated.
}


@end

File: 2017-02-23/SteviaExample/SteviaExample/SecondViewController.swift
//
//  SecondViewController.swift
//  SteviaExample
//
//  Created by Jason Liang on 2/23/17.
//  Copyright © 2017 Jason Liang. All rights reserved.
//

import UIKit

class SecondViewController: UIViewController {

  let baseView = SecondView()
  
  override func viewDidLoad() {
    super.viewDidLoad()
    self.view.sv(baseView)
    baseView.fillContainer()
  }

  override func didReceiveMemoryWarning() {
    super.didReceiveMemoryWarning()
    // Dispose of any resources that can be recreated.
  }


}

